great_expectations/exceptions/exceptions.py

```diff
@@ -225,42 +205,6 @@ class SuiteParameterError(GreatExpectationsError):
     pass
 
 
-class ProfilerError(GreatExpectationsError):
```

Comment:
Are you moving these errors to the rule_based_profiler package because they are no longer public?